docs/sources/alerting/configure-notifications/create-silence.md

```diff
@@ -21,6 +21,22 @@ labels:
     - oss
 title: Configure silences
 weight: 440
+refs:
```

Comment:
@jdbaldry @brendamuir 

These `refs` are necessary here for the `docs/shared` content, and they are duplicated in `notification-policies.md`.

Ideally, this could be set in the shared `md` file.